### ISSUE TITLE
Assertion error on header at data POST (waitress)

### ISSUE THREAD

**Issue by leplatrem** [spiral-project/daybed#45]
Request method: POST
Request URL: http://localhost:8000/data/mushroomspot
Request payload: {"mushroom":"tete","area":"[20.390625,7.536764322084078]"}

```
ERROR [waitress][Dummy-1] Exception when serving /data/mushroomspot
Traceback (most recent call last):
  File "env/local/lib/python2.7/site-packages/waitress/channel.py", line 329, in service
    task.service()
  File "env/local/lib/python2.7/site-packages/waitress/task.py", line 173, in service
    self.execute()
  File "env/local/lib/python2.7/site-packages/waitress/task.py", line 380, in execute
    app_iter = self.channel.server.application(env, start_response)
  File "env/local/lib/python2.7/site-packages/pyramid-1.4-py2.7.egg/pyramid/router.py", line 252, in __call__
    return response(request.environ, start_response)
  File "env/local/lib/python2.7/site-packages/WebOb-1.2.3-py2.7.egg/webob/response.py", line 1021, in __call__
    start_response(self.status, headerlist)
  File "env/local/lib/python2.7/site-packages/waitress/task.py", line 364, in start_response
    'Header value %r is not a string in %r' % (v, (k, v))
AssertionError: Header value u'http://localhost:8000/data/59fcf69f72069e8bf8d917a155001179' is not a string in ('location', u'http://localhost:8000/data/59fcf69f72069e8bf8d917a155001179')

```

**Comment by arnaudbos** [comment]
I'm having the same issue on master following the "todo list" example from the documentation.

I'm not familiar with waitress, is there something I need to do to fix this?

Edit: seems to be unicode related bug.

**Comment by arnaudbos** [comment]
Waitress only supports string headers it seems (https://github.com/Pylons/pyramid/blob/master/docs/whatsnew-1.3.rst), don't know about HTTP in general though (haven't checked).

Anyway, the error comes from here: https://github.com/spiral-project/daybed/blob/dev/daybed/views/data.py#L45 where `model_id` is of unicode type.

I think this could be fixed easily, I personally have this error at each data item POST with a nasty 500 error.

I can make a pull request if needed, just let me know :-)

Thanks!

**Comment by leplatrem** [comment]
Very good catch thanks !

Would definitely merge such a fix :)

**Comment by arnaudbos** [comment]
Done at #97

**Comment by almet** [comment]
Fixed in db98b01
